docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md

```diff
@@ -13,20 +13,33 @@ type: docs
 
 The YugabyteDB release that is powering a universe can be upgraded to get new features and fixes included in the release.
 
-Before starting the upgrade: 
+Before starting the upgrade:
 
-- Ensure that the YB Controller port 18018 is open on all YugabyteDB nodes. 
+- Ensure that the YB Controller port 18018 is open on all YugabyteDB nodes.
 
 - Consider importing a specific YugabyteDB release into YugabyteDB Anywhere, as follows:
 
   - Click the user profile icon and select **Releases**.
 
-  - Click **Import** to open the **Import Releases** dialog shown in the following illustration:<br>
+  - Click **Import** to open the **Import Releases** dialog shown in the following illustration:
 
-    ![Import Releases](/images/yp/import-releases.png)<br>
+    ![Import Releases](/images/yp/import-releases.png)
 
   - Specify the release version to import.
 
+    {{< note title="Manually-provisioned on-premises universe upgrades" >}}
+
+For v2.16 and later, YB Controller does not get installed during a YugabyteDB software upgrade of a manually provisioned [on-premises](../../configure-yugabyte-platform/set-up-cloud-provider/on-premises/) universe.
```

Comment:
@nbhatia-yb,, As we don't have the fix in 2.16.3, so Akki mentioned that we should have the workaround mentioned until we release 2.16.4.
We have a separate ticket for docs changes related to manually provisioned systemd managed universe [PLAT-8123]
Also, @aishwarya24 This workaround is for manually provisioned cron managed universes. We don' support ybc installation on manually provisioned systemd managed universe during software upgrade until we release 2.16.4 or 2.18.

Reply:
@nbhatia-yb @aishwarya24 This workaround is for all on-prem manually provisioned universes in 2.16.3. Also, this would not work if there are read replicas. We should probably be explicit about this in our latest 2.16 docs till we have our docs and bug fixes as part of 2.16.4 and 2.18.

Reply:
@vpatibandla-yb - I have made the note specific to the version 2.16.3 and have included it for preview(2.17) and stable(2.16) docs. Could you please take a look and let me know if I got it right.